src/app/clusters/push-av-stream-transport-server/push-av-stream-transport-server.h

```diff
@@ -0,0 +1,786 @@
+    {
+
+        chunkDuration = CMAFContainerOptions.chunkDuration;
+
+        if (CMAFContainerOptions.CENCKey.HasValue())
+        {
+            MutableByteSpan CENCKeyBuffer(mCENCKeyBuffer);
+            CopySpanToMutableSpan(CMAFContainerOptions.CENCKey.Value(), CENCKeyBuffer);
+            CENCKey.SetValue(CENCKeyBuffer);
+        }
+        else
+        {
+            CENCKey.ClearValue();
+        }
+
+        metadataEnabled = CMAFContainerOptions.metadataEnabled;
+
+        if (CMAFContainerOptions.CENCKey.HasValue())
+        {
+            MutableByteSpan CENCKeyIDBuffer(mCENCKeyIDBuffer);
+            CopySpanToMutableSpan(CMAFContainerOptions.CENCKeyID.Value(), CENCKeyIDBuffer);
+            CENCKeyID.SetValue(CENCKeyIDBuffer);
+        }
+        else
+        {
+            CENCKeyID.ClearValue();
+        }
+    }
+
+private:
+    uint8_t mCENCKeyBuffer[kMaxCENCKeyLength];
+    uint8_t mCENCKeyIDBuffer[kMaxCENCKeyIDLength];
+};
+
+struct ContainerOptionsStorage : public ContainerOptionsStruct
+{
+    ContainerOptionsStorage(){};
+
+    ContainerOptionsStorage(const ContainerOptionsStorage & aContainerOptionsStorage) { *this = aContainerOptionsStorage; }
+
+    ContainerOptionsStorage & operator=(const ContainerOptionsStorage & aContainerOptionsStorage)
+    {
+        containerType        = aContainerOptionsStorage.containerType;
+        CMAFContainerOptions = aContainerOptionsStorage.CMAFContainerOptions;
+
+        if (containerType == ContainerFormatEnum::kCmaf)
+        {
+            mCMAFContainerStorage = aContainerOptionsStorage.mCMAFContainerStorage;
+            CMAFContainerOptions.SetValue(mCMAFContainerStorage);
+        }
+
+        return *this;
+    }
+
+    ContainerOptionsStorage & operator=(const Structs::ContainerOptionsStruct::DecodableType aContainerOptions)
+    {
+        containerType = aContainerOptions.containerType;
+
+        if (containerType == ContainerFormatEnum::kCmaf)
+        {
+            mCMAFContainerStorage = aContainerOptions.CMAFContainerOptions.Value();
+            CMAFContainerOptions.SetValue(mCMAFContainerStorage);
+        }
+        else
+        {
+            CMAFContainerOptions.ClearValue();
+        }
+
+        return *this;
+    }
+
+    ContainerOptionsStorage(Structs::ContainerOptionsStruct::DecodableType containerOptions)
+    {
+        containerType = containerOptions.containerType;
+
+        if (containerType == ContainerFormatEnum::kCmaf)
+        {
+            mCMAFContainerStorage = containerOptions.CMAFContainerOptions.Value();
+            CMAFContainerOptions.SetValue(mCMAFContainerStorage);
+        }
+        else
+        {
+            CMAFContainerOptions.ClearValue();
+        }
+    }
+
+private:
+    CMAFContainerOptionsStorage mCMAFContainerStorage;
+};
+
+struct TransportOptionsStorage : public TransportOptionsStruct
+{
+    TransportOptionsStorage(){};
```

Comment:
```suggestion
    TransportOptionsStorage(){}
```

Check throughout for extra `;`

Reply:
Removed them.